src/stories/Form.js

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react'
+import { storiesOf, action } from '@storybook/react'
+import { Box, Flex, Txt } from '../'
+import { Form } from '../unstable/'
+
+const basicPokedexSchema = {
+  type: 'object',
+  properties: {
+    Name: {
+      type: 'string'
+    },
+    Height: {
+      type: 'number'
+    },
+    Weight: {
+      type: 'number'
+    },
+    Description: {
+      type: 'string'
+    },
+    Category: {
+      type: 'string'
+    },
+    Abilities: {
+      type: 'string'
+    },
+    pokedex_number: {
+      title: 'National Pokedex Number',
+      type: 'number'
+    },
+    caught: {
+      type: 'boolean'
+    },
+    first_seen: {
+      title: 'First seen',
+      type: 'string',
+      format: 'date-time'
+    }
+  }
+}
+
+class FormDemo extends React.Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      formData: {}
+    }
+  }
+
+  change ({ formData }) {
+    action('Form.onChange')(formData)
+    this.setState({ formData })
+  }
+
+  submit (data) {
+    action('Form.onSubmit')(data)
+  }
+
+  render () {
+    return (
+      <Flex>
+        <Box flex='1' m={30}>
+          <Form
+            onChange={d => this.change(d)}
+            onSubmit={d => this.submit(d)}
+            value={this.state.formData}
+            {...this.props}
+          />
+        </Box>
+        <Box flex='1' p={30}>
+          <Txt monospace>
+            <pre>{JSON.stringify(this.state.formData, null, 4)}</pre>
+          </Txt>
+        </Box>
+      </Flex>
+    )
+  }
+}
+
+storiesOf('[beta] Form', module)
```

Comment:
If-you-are-in-the-mood: How about moving the `[beta]` past to the end so that the it doesn't look like a mis-ordering in the stories menu
![image](https://user-images.githubusercontent.com/1295829/39179225-72ad716c-47bc-11e8-854c-3060a107e022.png)

Or someone could argue about a whole different ordering for all the beta components (eg last), but probably doesn't worth it.